core/src/test/scripts/test_data/data_structural_variants_missing_values.txt

```diff
@@ -1,8 +1,8 @@
-TCGA-A2-A04P-01	NA	KIAA1549	ENST00000242365	15	7	138536968	KIAA1549-BRAF.K16B10.COSF509_1	NA	BRAF		10	7	140482957	KIAA1549-BRAF.K16B10.COSF509_2	NA	GRCh37	no	yes	NA	1000	NA	900	NA	NA	NA	NA	KIAA1549-BRAF.K16B10.COSF509	NA	NA	NA	Fusion	NA	NA	Gain-of-Function	COSMIC:COSF509	NA	NA	NA	NA
-TCGA-A2-A04P-01	NA	NCOA4		7	10	51582939	NCOA4-RET.N7R12_1	NA	RET	ENST00000340058	12	10	43612031	NCOA4-RET.N7R12_2	NA	GRCh37	no	yes	NA	1001	NA	800	NA	NA	NA	NA	NCOA4-RET.N7R1	NA	NA	NA	Fusion	NA	NA	Gain-of-Function	NA	NA	NA	NA	NA
-TCGA-A2-A04P-01	NA	EML4	ENST00000318522	6	2	42492091	EML4-ALK.E6bA20.AB374362_1	NA	ALK	ENST00000389048	20	2	29446394	EML4-ALK.E6bA20.AB374362_2	NA	GRCh37	no	yes	NA	1002	NA	700	NA	NA	NA	NA	EML4-ALK.E6bA20.AB374362	NA	NA	NA	Fusion	NA	NA	Gain-of-Function	GENBANK:AB374362	NA	NA	NA	NA
-TCGA-A2-A04P-01	NA	TMPRSS2	ENST00000332149	1	21	42880007	TMPRSS2-ERG.T1E2.COSF23.1_1	NA	ERG	ENST00000442448	2	21	39956869	TMPRSS2-ERG.T1E2.COSF23.1_2	NA	GRCh37	no	yes	NA	1003	NA	600	NA	NA	NA	NA	TMPRSS2-ERG.T1E2.COSF23.1	NA	NA	NA	Fusion	NA	NA	Gain-of-Function	COSMIC:COSF23	NA	NA	NA	NA
-TCGA-A2-A04P-01	NA	EGFR	ENST00000275493		7	55087058	EGFR-EGFR.E1E8.DelPositive.1_1	NA	EGFR	ENST00000275493	8	7	55223522	EGFR-EGFR.E1E8.DelPositive.1_2	NA	GRCh37	no	yes	NA	1004	NA	500	NA	NA	NA	NA	EGFR-EGFR.E1E8.DelPositive	NA	NA	NA	Fusion	NA	NA	NA	NA	NA	NA	NA	NA
-TCGA-A2-A04P-01	NA	ALK	ENST00000389048	11	2	29497964	ALK-PTPN3.A11P3_1	NA	PTPN3	ENST00000374541	3	9	112219679	ALK-PTPN3.A11P3_2	NA	GRCh37	no	yes	NA	1005	NA	400	NA	NA	NA	NA	ALK-PTPN3.A11P3	NA	NA	NA	Fusion	NA	NA	NA	NA	NA	NA	NA	NA
-TCGA-A1-A0SB-01	NA	EML4	ENST00000318522	13	2	42522656	EML4-ALK.E13A20.AB462411_1	NA	ALK	ENST00000389048		2	29446335	EML4-ALK.E13A20.AB462411_2	NA	GRCh37	no	yes	NA	1006	NA	300	NA	NA	NA	NA	EML4-ALK.E13A20	NA	NA	NA	Fusion	NA	NA	Gain-of-Function	GENBANK:AB462411	NA	NA	NA	NA
+Sample_Id	Site1_Entrez_Gene_Id	Site1_Hugo_Symbol	Site1_Ensembl_Transcript_Id	Site1_Region_Number	Site1_Chromosome	Site1_Position	Site1_Description	Site2_Entrez_Gene_Id	Site2_Hugo_Symbol	Site2_Ensembl_Transcript_Id	Site2_Region_Number	Site2_Chromosome	Site2_Position	Site2_Description	Site2_Effect_On_Frame	NCBI_Build	DNA_Support	RNA_Support	Normal_Read_Count	Tumor_Read_Count	Normal_Variant_Count	Tumor_Variant_Count	Normal_Paired_End_Read_Count	Tumor_Paired_End_Read_Count	Normal_Split_Read_Count	Tumor_Split_Read_Count	Annotation	Breakpoint_Type	Center	Connection_Type	Event_Info	Class	SV_Length	Comments	External_Annotation	cbp_driver	cbp_driver_annotation	cbp_driver_tiers	cbp_driver_tiers_annotation	SV_Status
```

Comment:
SV_Length is just Length I

Reply:
also not sure if i tmatters but I don't see a Site1/2_Region - dunno if this file is supposed to encompass all fields

Reply:
Isn't it still SV_Length?

Reply:
This file is referred to in the test https://github.com/cBioPortal/cbioportal/blob/0c9bf854bc700e5dc0e8bc6c391a8d6e9c373755/core/src/test/scripts/unit_tests_validate_data.py#L1769 We are only testing whether the entrez gene id and hugo symbols are found in both the sites here. I guess the file doesn't have to encompass all the fields?

Reply:
Looked at the backend PR is looks like its Length